auth-web/src/services/keycloak.services.ts

```diff
@@ -20,13 +21,17 @@ export default {
     return this.kc.init({ token: token, onLoad: 'login-required' })
   },
 
-  initSessionStorage () {
+  initSession () {
     configHelper.addToSession(SessionStorageKeys.KeyCloakToken, this.kc.token)
     configHelper.addToSession(SessionStorageKeys.KeyCloakRefreshToken, this.kc.refreshToken)
+    this.parsedToken = this.kc.tokenParsed
   },
 
   getUserInfo () : UserInfo {
-    let token = this.kc.tokenParsed
+    let token = this.parsedToken
```

Comment:
may be the token variable is not needed?  this.parsedToken can be directly used?

Reply:
Changed made in latest commit